portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -11,13 +12,20 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.Query.SortDirection;
-import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.FetchOptions;
 import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 import com.google.sps.objects.User;
 
+/**
+ * A class to fetch data from the datastore library.
+ * @param userData The current data that has been fetched from the datastore
+ *     library, it acts as a local cache of the data that has been recorded.
+ *     Both the doGet and doPost methods rely on this local cache, only
+ *     refreshing if the refresh parameter is specified and true.
```

Comment:
I think the description of `userData` should go in its own comment right above the actual variable, not in the class comment.